### PR TITLE
[TEST] - increased unit test coverage

### PR DESCRIPTION
**Pull request by krivera-pentaho**

### CODE REVIEW

**Comment by krivera-pentaho** [comment]
@lucboudreau please review

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,8 @@ extensions/test-src/solution/test/tmp/
 build-utils/dist
 build-utils/lib
 build-utils/test-lib
-.settings
\ No newline at end of file
+.settings
+.DS_Store
+override.properties
+build-res/
```
Comment (lucboudreau) [inline .gitignore]: Not sure about this one (build-res). we don't want git to ignore it, jsut developers not to change it. Adding it here will mess up things for the automated scripts that update these files automatically.

File: user-console/source/org/pentaho/mantle/client/admin/ContentCleanerPanel.java [hunk 2]
```diff
@@ -49,6 +49,7 @@
 import org.pentaho.mantle.client.messages.Messages;
 import org.pentaho.mantle.client.workspace.JsJob;
 import org.pentaho.mantle.client.workspace.JsJobParam;
+import com.google.gwt.user.client.rpc.AsyncCallback;
```
Comment (lucboudreau) [inline user-console/source/org/pentaho/mantle/client/admin/ContentCleanerPanel.java]: Doesn't look needed.

**Comment by lucboudreau** [comment]
This fails to compile.

http://10.100.11.222:8080/job/wingman-ant/111/console

**Comment by krivera-pentaho** [comment]
@lucboudreau, I made changes based on your suggestions. It resolves, compiles, and all tests pass in my workspace, but I am experiencing a similar issue as the wingman server when I run "ant jacoco". I am going to investigate and see why it is not including my Mockito packages.

**Comment by lucboudreau** [comment]
Let me know when you find something, I'll test it for you, no problem.

**Comment by krivera-pentaho** [comment]
@lucboudreau, i added the missing dependencies so that the "ant jacoco" runs.

**Comment by lucboudreau** [comment]
Build Commands:
==============
Build:
------
1. rm -r ~/.kettle; rm -r ~/.pentaho; echo removed hidden folders
2. ant -Djunit.jvmargs=-XX:PermSize=256m -f user-console/build.xml clean-all resolve jacoco jacoco-integration checkstyle publish-local

Cleanup:
-------
rm -r ~/.ivy2/local || echo no publish local to remove

Changed files:
==============
  .gitignore
  user-console/build.xml
  user-console/ivy.xml
  user-console/mantle.iml
  user-console/source/org/pentaho/mantle/client/MantleApplication.java
  user-console/source/org/pentaho/mantle/client/MantleEntryPoint.java
  user-console/source/org/pentaho/mantle/client/MantleMenuBar.java
  user-console/source/org/pentaho/mantle/client/admin/ChangePasswordDialog.java
  user-console/source/org/pentaho/mantle/client/admin/ContentCleanerPanel.java
  user-console/source/org/pentaho/mantle/client/admin/EmailAdminPanelController.java
  user-console/source/org/pentaho/mantle/client/admin/PermissionsPanel.java
  user-console/source/org/pentaho/mantle/client/admin/RoleDialog.java
  user-console/source/org/pentaho/mantle/client/admin/UserDialog.java
  user-console/source/org/pentaho/mantle/client/admin/UserRolesAdminPanelController.java
  user-console/source/org/pentaho/mantle/client/commands/AboutCommand.java
  user-console/source/org/pentaho/mantle/client/commands/CheckForSoftwareUpdatesCommand.java
  user-console/source/org/pentaho/mantle/client/commands/CollapseBrowserCommand.java
  user-console/source/org/pentaho/mantle/client/commands/DeleteFileCommand.java
  user-console/source/org/pentaho/mantle/client/commands/PasteFilesCommand.java
  user-console/source/org/pentaho/mantle/client/commands/RestoreFileCommand.java
  user-console/test-src/org/pentaho/mantle/client/EmptyRequestCallbackTest.java
  user-console/test-src/org/pentaho/mantle/client/MantleApplicationTest.java
  user-console/test-src/org/pentaho/mantle/client/MantleEntryPointTest.java
  user-console/test-src/org/pentaho/mantle/client/NothingTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/ChangePasswordDialogTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/ContentCleanerPanelTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/EmailTesterTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/PermissionsPanelTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/RoleDialogTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/UserDialogTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/UserRolesAdminPanelControllerTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/UserRolesAdminPanelTest.java
  user-console/test-src/org/pentaho/mantle/client/commands/AboutCommandTest.java
  user-console/test-src/org/pentaho/mantle/client/commands/CheckForSoftwareUpdatesCommandTest.java
  user-console/test-src/org/pentaho/mantle/client/commands/CollapseBrowserCommandTest.java
  user-console/test-src/org/pentaho/mantle/client/commands/DeleteFileCommandTest.java
  user-console/test-src/org/pentaho/mantle/client/commands/PasteFilesCommandTest.java
  user-console/test-src/org/pentaho/mantle/client/commands/RestoreFileCommandTest.java
  user-console/user-console.iml

Checkstyle violations:
======================
File user-console/source/org/pentaho/mantle/client/MantleApplication.java
1. Message: '(' is not followed by whitespace. at Line: 102 Column: 29
2. Message: ')' is not preceded with whitespace. at Line: 102 Column: 45
3. Message: ')' is not preceded with whitespace. at Line: 104 Column: 13
4. Message: '(' is not followed by whitespace. at Line: 160 Column: 29
5. Message: ')' is not preceded with whitespace. at Line: 160 Column: 42
6. Message: '(' is not followed by whitespace. at Line: 164 Column: 28
7. Message: ')' is not preceded with whitespace. at Line: 164 Column: 52
8. Message: '(' is not followed by whitespace. at Line: 232 Column: 25
9. Message: ')' is not preceded with whitespace. at Line: 232 Column: 50
10. Message: '(' is not followed by whitespace. at Line: 236 Column: 41
11. Message: '(' is not followed by whitespace. at Line: 236 Column: 69
12. Message: ')' is not preceded with whitespace. at Line: 236 Column: 72
13. Message: ')' is not preceded with whitespace. at Line: 236 Column: 73
14. Message: '(' is not followed by whitespace. at Line: 252 Column: 50
15. Message: '(' is not followed by whitespace. at Line: 253 Column: 29
16. Message: ')' is not preceded with whitespace. at Line: 253 Column: 67
17. Message: '(' is not followed by whitespace. at Line: 256 Column: 13
18. Message: ')' is not preceded with whitespace. at Line: 256 Column: 33
19. Message: '(' is not followed by whitespace. at Line: 257 Column: 16
20. Message: ')' is not preceded with whitespace. at Line: 257 Column: 57
21. Message: '(' is not followed by whitespace. at Line: 258 Column: 50
22. Message: ')' is not preceded with whitespace. at Line: 258 Column: 51
23. Message: '(' is not followed by whitespace. at Line: 259 Column: 32
24. Message: ')' is not preceded with whitespace. at Line: 259 Column: 69
25. Message: '(' is not followed by whitespace. at Line: 262 Column: 32
26. Message: '(' is not followed by whitespace. at Line: 262 Column: 62
27. Message: ')' is not preceded with whitespace. at Line: 262 Column: 76
28. Message: ')' is not preceded with whitespace. at Line: 262 Column: 77
29. Message: '(' is not followed by whitespace. at Line: 265 Column: 29
30. Message: ')' is not preceded with whitespace. at Line: 265 Column: 45
31. Message: ')' is not preceded with whitespace. at Line: 267 Column: 13
32. Message: '(' is not followed by whitespace. at Line: 274 Column: 37
33. Message: ')' is not preceded with whitespace. at Line: 274 Column: 47
34. Message: '(' is not followed by whitespace. at Line: 275 Column: 41
35. Message: ')' is not preceded with whitespace. at Line: 275 Column: 44
36. Message: '(' is not followed by whitespace. at Line: 278 Column: 28
37. Message: ')' is not preceded with whitespace. at Line: 278 Column: 38
38. Message: 'method def modifier' have incorrect indentation level 4, expected level should be 2. at Line: 278 Column: None
39. Message: '(' is not followed by whitespace. at Line: 279 Column: 9
40. Message: ')' is not preceded with whitespace. at Line: 279 Column: 20
41. Message: '(' is not followed by whitespace. at Line: 280 Column: 28
42. Message: ')' is not preceded with whitespace. at Line: 280 Column: 31
43. Message: '(' is not followed by whitespace. at Line: 306 Column: 36
44. Message: ')' is not preceded with whitespace. at Line: 306 Column: 76
45. Message: '(' is not followed by whitespace. at Line: 307 Column: 31
46. Message: ')' is not preceded with whitespace. at Line: 307 Column: 42
47. Message: '(' is not followed by whitespace. at Line: 310 Column: 40
48. Message: ')' is not preceded with whitespace. at Line: 310 Column: 113
49. Message: ')' is not preceded with whitespace. at Line: 311 Column: 69
50. Message: '(' is not followed by whitespace. at Line: 318 Column: 173
51. Message: '(' is not followed by whitespace. at Line: 318 Column: 79
52. Message: ')' is not preceded with whitespace. at Line: 318 Column: 100
53. Message: ')' is not preceded with whitespace. at Line: 318 Column: 194
54. Message: '(' is not followed by whitespace. at Line: 320 Column: 149
55. Message: '(' is not followed by whitespace. at Line: 320 Column: 57
56. Message: ')' is not preceded with whitespace. at Line: 320 Column: 169
57. Message: ')' is not preceded with whitespace. at Line: 320 Column: 77
58. Message: '(' is not followed by whitespace. at Line: 324 Column: 18
59. Message: ')' is not preceded with whitespace. at Line: 324 Column: 30
60. Message: '(' is not followed by whitespace. at Line: 325 Column: 18
61. Message: ')' is not preceded with whitespace. at Line: 325 Column: 35
62. Message: '(' is not followed by whitespace. at Line: 326 Column: 18
63. Message: ')' is not preceded with whitespace. at Line: 326 Column: 30
64. Message: '(' is not followed by whitespace. at Line: 327 Column: 18
65. Message: ')' is not preceded with whitespace. at Line: 327 Column: 35
66. Message: '(' is not followed by whitespace. at Line: 330 Column: 20
67. Message: ')' is not preceded with whitespace. at Line: 330 Column: 31
68. Message: '(' is not followed by whitespace. at Line: 331 Column: 20
69. Message: ')' is not preceded with whitespace. at Line: 331 Column: 32
70. Message: '(' is not followed by whitespace. at Line: 355 Column: 18
71. Message: '(' is not followed by whitespace. at Line: 355 Column: 28
72. Message: ')' is not preceded with whitespace. at Line: 355 Column: 22
73. Message: ')' is not preceded with whitespace. at Line: 355 Column: 51
74. Message: '(' is not followed by whitespace. at Line: 363 Column: 18
75. Message: ')' is not preceded with whitespace. at Line: 363 Column: 22
76. Message: '(' is not followed by whitespace. at Line: 370 Column: 57
77. Message: ')' is not preceded with whitespace. at Line: 370 Column: 84
78. Message: '(' is not followed by whitespace. at Line: 97 Column: 46
79. Message: '(' is not followed by whitespace. at Line: 98 Column: 29
80. Message: ')' is not preceded with whitespace. at Line: 98 Column: 56
81. Message: '(' is not followed by whitespace. at Line: 99 Column: 30
82. Message: '(' is not followed by whitespace. at Line: 99 Column: 58
83. Message: ')' is not preceded with whitespace. at Line: 99 Column: 66
84. Message: ')' is not preceded with whitespace. at Line: 99 Column: 67

File user-console/source/org/pentaho/mantle/client/MantleEntryPoint.java
1. Message: '(' is not followed by whitespace. at Line: 101 Column: 26
2. Message: ')' is not preceded with whitespace. at Line: 101 Column: 42
3. Message: '(' is not followed by whitespace. at Line: 85 Column: 31
4. Message: ')' is not preceded with whitespace. at Line: 85 Column: 43
5. Message: '(' is not followed by whitespace. at Line: 86 Column: 41
6. Message: ')' is not preceded with whitespace. at Line: 86 Column: 46
7. Message: '(' is not followed by whitespace. at Line: 93 Column: 36
8. Message: ')' is not preceded with whitespace. at Line: 93 Column: 76

File user-console/source/org/pentaho/mantle/client/admin/ContentCleanerPanel.java
1. Message: '(' is not followed by whitespace. at Line: 73 Column: 36
2. Message: ')' is not preceded with whitespace. at Line: 73 Column: 76
3. Message: '(' is not followed by whitespace. at Line: 74 Column: 31
4. Message: ')' is not preceded with whitespace. at Line: 74 Column: 42

File user-console/source/org/pentaho/mantle/client/admin/EmailAdminPanelController.java
1. Message: ',' is preceded with whitespace. at Line: 118 Column: 14
2. Message: ',' is preceded with whitespace. at Line: 174 Column: 14
3. Message: ',' is preceded with whitespace. at Line: 283 Column: 10
4. Message: ',' is preceded with whitespace. at Line: 284 Column: 10
5. Message: ',' is preceded with whitespace. at Line: 295 Column: 16
6. Message: 'method def modifier' have incorrect indentation level 4, expected level should be 2. at Line: 53 Column: None

File user-console/source/org/pentaho/mantle/client/admin/UserRolesAdminPanelController.java
1. Message: ',' is preceded with whitespace. at Line: 154 Column: 10
2. Message: ',' is preceded with whitespace. at Line: 166 Column: 8
3. Message: Using the '.*' form of import should be avoided - com.google.gwt.event.dom.client.*. at Line: 22 Column: None
4. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 23 Column: None
5. Message: ',' is preceded with whitespace. at Line: 253 Column: 6
6. Message: '(' is not followed by whitespace. at Line: 531 Column: 9
7. Message: ',' is preceded with whitespace. at Line: 699 Column: 12
8. Message: ',' is preceded with whitespace. at Line: 700 Column: 12
9. Message: ',' is preceded with whitespace. at Line: 727 Column: 12
10. Message: ',' is preceded with whitespace. at Line: 728 Column: 12

File user-console/source/org/pentaho/mantle/client/commands/AboutCommand.java
1. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 21 Column: None

File user-console/source/org/pentaho/mantle/client/commands/CheckForSoftwareUpdatesCommand.java
1. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 21 Column: None

File user-console/source/org/pentaho/mantle/client/commands/DeleteFileCommand.java
1. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 21 Column: None

File user-console/source/org/pentaho/mantle/client/commands/PasteFilesCommand.java
1. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 21 Column: None

File user-console/source/org/pentaho/mantle/client/commands/RestoreFileCommand.java
1. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 21 Column: None

File user-console/test-src/org/pentaho/mantle/client/admin/UserRolesAdminPanelTest.java
1. Message: '(' is not followed by whitespace. at Line: 14 Column: 19
2. Message: ')' is not preceded with whitespace. at Line: 14 Column: 52
3. Message: '(' is not followed by whitespace. at Line: 9 Column: 10
4. Message: ')' is not preceded with whitespace. at Line: 9 Column: 36

File user-console/test-src/org/pentaho/mantle/client/commands/AboutCommandTest.java
1. Message: '(' is not followed by whitespace. at Line: 30 Column: 46
2. Message: ')' is not preceded with whitespace. at Line: 30 Column: 66
3. Message: ')' is not preceded with whitespace. at Line: 36 Column: 31
4. Message: ')' is not preceded with whitespace. at Line: 37 Column: 31
5. Message: ')' is not preceded with whitespace. at Line: 38 Column: 31


License header violations:
==========================
1. user-console/test-src/org/pentaho/mantle/client/EmptyRequestCallbackTest.java
2. user-console/test-src/org/pentaho/mantle/client/MantleApplicationTest.java
3. user-console/test-src/org/pentaho/mantle/client/MantleEntryPointTest.java
4. user-console/test-src/org/pentaho/mantle/client/admin/ChangePasswordDialogTest.java
5. user-console/test-src/org/pentaho/mantle/client/admin/ContentCleanerPanelTest.java
6. user-console/test-src/org/pentaho/mantle/client/admin/EmailTesterTest.java
7. user-console/test-src/org/pentaho/mantle/client/admin/PermissionsPanelTest.java
8. user-console/test-src/org/pentaho/mantle/client/admin/RoleDialogTest.java
9. user-console/test-src/org/pentaho/mantle/client/admin/UserDialogTest.java
10. user-console/test-src/org/pentaho/mantle/client/admin/UserRolesAdminPanelControllerTest.java
11. user-console/test-src/org/pentaho/mantle/client/admin/UserRolesAdminPanelTest.java
12. user-console/test-src/org/pentaho/mantle/client/commands/AboutCommandTest.java
13. user-console/test-src/org/pentaho/mantle/client/commands/CheckForSoftwareUpdatesCommandTest.java
14. user-console/test-src/org/pentaho/mantle/client/commands/CollapseBrowserCommandTest.java
15. user-console/test-src/org/pentaho/mantle/client/commands/DeleteFileCommandTest.java
16. user-console/test-src/org/pentaho/mantle/client/commands/PasteFilesCommandTest.java
17. user-console/test-src/org/pentaho/mantle/client/commands/RestoreFileCommandTest.java







Unit test coverage change:
============
org.pentaho.mantle.client.EmptyRequestCallback:
 * linePercentCoverage :66.67%
 * methodPercentCoverage :60.00%
 * complexityPercentCoverage :60.00%
 * instructionPercentCoverage :83.33%

**Comment by lucboudreau** [comment]
Checkstyle and licenses to fix. Please also squash these commits into a single one.

**Comment by krivera-pentaho** [comment]
@lucboudreau commit squashed, checkstyles run on all edited files.

**Comment by krivera-pentaho** [comment]
@lucboudreau please review

**Comment by lucboudreau** [comment]
Build Commands:
==============
Build:
------
1. rm -r ~/.kettle; rm -r ~/.pentaho; echo removed hidden folders
2. ant -Djunit.jvmargs=-XX:PermSize=256m -f user-console/build.xml clean-all resolve jacoco jacoco-integration checkstyle publish-local

Cleanup:
-------
rm -r ~/.ivy2/local || echo no publish local to remove

Changed files:
==============
  .gitignore
  user-console/build.xml
  user-console/ivy.xml
  user-console/mantle.iml
  user-console/source/org/pentaho/mantle/client/MantleApplication.java
  user-console/source/org/pentaho/mantle/client/MantleEntryPoint.java
  user-console/source/org/pentaho/mantle/client/MantleMenuBar.java
  user-console/source/org/pentaho/mantle/client/admin/ChangePasswordDialog.java
  user-console/source/org/pentaho/mantle/client/admin/ContentCleanerPanel.java
  user-console/source/org/pentaho/mantle/client/admin/EmailAdminPanelController.java
  user-console/source/org/pentaho/mantle/client/admin/PermissionsPanel.java
  user-console/source/org/pentaho/mantle/client/admin/RoleDialog.java
  user-console/source/org/pentaho/mantle/client/admin/UserDialog.java
  user-console/source/org/pentaho/mantle/client/admin/UserRolesAdminPanelController.java
  user-console/source/org/pentaho/mantle/client/commands/AboutCommand.java
  user-console/source/org/pentaho/mantle/client/commands/CheckForSoftwareUpdatesCommand.java
  user-console/source/org/pentaho/mantle/client/commands/CollapseBrowserCommand.java
  user-console/source/org/pentaho/mantle/client/commands/DeleteFileCommand.java
  user-console/source/org/pentaho/mantle/client/commands/PasteFilesCommand.java
  user-console/source/org/pentaho/mantle/client/commands/RestoreFileCommand.java
  user-console/test-src/org/pentaho/mantle/client/EmptyRequestCallbackTest.java
  user-console/test-src/org/pentaho/mantle/client/MantleApplicationTest.java
  user-console/test-src/org/pentaho/mantle/client/MantleEntryPointTest.java
  user-console/test-src/org/pentaho/mantle/client/NothingTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/ChangePasswordDialogTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/ContentCleanerPanelTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/EmailTesterTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/PermissionsPanelTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/RoleDialogTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/UserDialogTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/UserRolesAdminPanelControllerTest.java
  user-console/test-src/org/pentaho/mantle/client/admin/UserRolesAdminPanelTest.java
  user-console/test-src/org/pentaho/mantle/client/commands/AboutCommandTest.java
  user-console/test-src/org/pentaho/mantle/client/commands/CheckForSoftwareUpdatesCommandTest.java
  user-console/test-src/org/pentaho/mantle/client/commands/CollapseBrowserCommandTest.java
  user-console/test-src/org/pentaho/mantle/client/commands/DeleteFileCommandTest.java
  user-console/test-src/org/pentaho/mantle/client/commands/PasteFilesCommandTest.java
  user-console/test-src/org/pentaho/mantle/client/commands/RestoreFileCommandTest.java
  user-console/user-console.iml

Checkstyle violations:
======================
File user-console/source/org/pentaho/mantle/client/MantleApplication.java
1. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 25 Column: None
2. Message: '?' should be on a new line. at Line: 308 Column: 106
3. Message: '?' should be on a new line. at Line: 312 Column: 83
4. Message: Using the '.*' form of import should be avoided - org.pentaho.mantle.client.events.*. at Line: 40 Column: None

File user-console/source/org/pentaho/mantle/client/MantleEntryPoint.java
1. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 22 Column: None

File user-console/source/org/pentaho/mantle/client/admin/ContentCleanerPanel.java
1. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 28 Column: None
2. Message: Using the '.*' form of import should be avoided - com.google.gwt.user.client.ui.*. at Line: 35 Column: None

File user-console/source/org/pentaho/mantle/client/admin/EmailAdminPanelController.java
1. Message: Using the '.*' form of import should be avoided - com.google.gwt.event.dom.client.*. at Line: 21 Column: None
2. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 24 Column: None

File user-console/source/org/pentaho/mantle/client/admin/UserRolesAdminPanelController.java
1. Message: Using the '.*' form of import should be avoided - com.google.gwt.event.dom.client.*. at Line: 22 Column: None
2. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 23 Column: None

File user-console/source/org/pentaho/mantle/client/commands/AboutCommand.java
1. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 21 Column: None

File user-console/source/org/pentaho/mantle/client/commands/CheckForSoftwareUpdatesCommand.java
1. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 21 Column: None

File user-console/source/org/pentaho/mantle/client/commands/DeleteFileCommand.java
1. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 21 Column: None

File user-console/source/org/pentaho/mantle/client/commands/PasteFilesCommand.java
1. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 21 Column: None

File user-console/source/org/pentaho/mantle/client/commands/RestoreFileCommand.java
1. Message: Using the '.*' form of import should be avoided - com.google.gwt.http.client.*. at Line: 21 Column: None


License header violations:
==========================
No violations found








Unit test coverage change:
============
org.pentaho.mantle.client.EmptyRequestCallback:
 * linePercentCoverage :66.67%
 * methodPercentCoverage :60.00%
 * complexityPercentCoverage :60.00%
 * instructionPercentCoverage :83.33%

**Comment by krivera-pentaho** [comment]
I will run checkstyles on those files, again, but i am not getting those errors. Do i need to updated my checkstyles xml?

**Comment by lucboudreau** [comment]
Possibly. We'll be looking at consolidating the checkstyle rules this week.

**Pull request closed without merging**